testsuite/features/srv_spacewalk_remove_channel_tool_delete_channel.feature

```diff
@@ -0,0 +1,46 @@
+# COPYRIGHT (c) 2019 SUSE LLC
+# Licensed under the terms of the MIT License.
+
+Feature: Delete channels with clone using spacewalk-remove-channel tool
+  We cannot delete a channel if it has a clone
+
+  Scenario: Clone the first channel
+    Given I am on the manage software channels page
+    When I follow "Clone Channel"
+    And I select "Test-Channel-x86_64" as the origin channel
+    And I click on "Clone Channel"
+    Then I should see a "Create Software Channel" text
+    And I should see a "Current state of the channel" text
+    When I click on "Clone Channel"
+    Then I should see a "Clone of Test-Channel-x86_64" text
+
+  Scenario: Clone a second channel using first channel as base
+    Given I am on the manage software channels page
+    When I follow "Clone Channel"
+    And I select "Clone of Test-Channel-x86_64" as the origin channel
+    And I click on "Clone Channel"
+    Then I should see a "Create Software Channel" text
+    And I should see a "Current state of the channel" text
+    When I click on "Clone Channel"
+    Then I should see a "Clone of Clone of Test-Channel-x86_64" text
+
+  Scenario: Verify if both clone channels exists
+    When I list channels with spacewalk-remove-channel
+    Then I should see "clone-test-channel-x86_64" in the output
+    And I should see "clone-clone-test-channel-x86_64" in the output
+
+  Scenario: Delete channel with one clone
+    When I delete these channels with spacewalk-remove-channel:
+      |clone-test-channel-x86_64|
+    Then I should see "Error: cannot remove channel" in the output
+    And  I should see "clone channel(s) exist" in the output
+    And  I should see "clone-test-channel-x86_64" in the output
+    And  I should see "clone-clone-test-channel-x86_64" in the output
```

Comment:
```suggestion
    And  I should get "clone-clone-test-channel-x86_64"
```

Reply:
Yeah, it exists:
```
Then(/^I should see "(.*?)" in the output$/) do |arg1|
  assert_includes(@command_output, arg1)
end
```
But it looks also like a duplicated step for me. Or I could miss something?

Reply:
yes that looks like a duplicated method but I'm not using it anymore in the tests I add. I change the test to use the step you suggested. Thanks